git/pre-commit

```diff
@@ -2,6 +2,16 @@
 
 if [ "$NOLINT" ]; then exit 0; fi
 
+if [ ! $TRAVIS = true ]
```

Comment:
Sorry, that second link is not working. I'm from mobile. Here: https://goo.gl/KNLjiE